graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/model/SubscriptionOperationMessage.kt

```diff
@@ -42,7 +42,6 @@ data class SubscriptionOperationMessage(
         GQL_CONNECTION_ERROR("connection_error"),
         GQL_DATA("data"),
         GQL_ERROR("error"),
-        GQL_COMPLETE("complete"),
```

Comment:
We still need to return this message to be compliant with the graphql-ws spec

Reply:
I wasn't aware of https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md I'll try to change so that on u subscribe it's still send.

Reply:
I return it now on a succesfull stop operation. Also added test for it.